src/pybind/mgr/cephadm/module.py

```diff
@@ -2275,11 +2286,13 @@ def _ok_to_stop(remove_daemon_hosts: Set[orchestrator.DaemonDescription]) -> boo
             # let's find a subset that is ok-to-stop
             remove_daemon_hosts.pop()
         for d in remove_daemon_hosts:
+            r = True
             # NOTE: we are passing the 'force' flag here, which means
             # we can delete a mon instances data.
             self._remove_daemon(d.name(), d.hostname)
```

Comment:
It's possible, although much less expected.  I wasn't trying to improve the remove case because I'd not experienced any issues there, but nearly anything *can* fail, it's true.